### ISSUE TITLE
Error trying to run install script

### ISSUE THREAD

**Issue by teredactle** [FiveLeavesLeft/repo#17]
Trying to install this on a WyzeCam3 [wyze_hacks_0_5_06.zip]
Ubuntu 18.04
Made sure Python3 installed
Made sure python-pip installed

When I try to run ./remote_install.sh I get this.

Found local config file, including into firmware update archive...
Upgrade/config.inc
/usr/bin/python3: No module named pip
Traceback (most recent call last):
  File "./wyze_updater.py", line 294, in <module>
    help='Supported actions')
  File "/usr/lib/python3.6/argparse.py", line 1716, in add_subparsers
    action = parsers_class(option_strings=[], **kwargs)
TypeError: __init__() got an unexpected keyword argument 'required'

Help would be appreicated

**Comment by FiveLeavesLeft** [comment]
I think you need python 3.8. Also, you'll need the module "pip" installed.

**Comment by mrlt8** [comment]
> Made sure python-pip installed

IIRC, python-pip is python 2, you need to install python3-pip.

**Comment by endertable** [comment]
Based on the error, your python interpreter is 3.6 and you should use 3.7 or higher

**Comment by teredactle** [comment]
Thanks for this, installed 3.8 as well as python3-pip and now it's "installing".
I also have fw 228 on the WC3 which the docs say it's supported.

Once I get the prompt and put select y for the camera, it goes on down the list and I hit N for the rest.

Then it just has the screen with dots that goes on forever....................................................
......................................... and so on.

I don't hear "installation begins" or "installation complete".... how long do I need to wait?
I see that the light on the camera is purple during all of this, sort of flickering. 
Cannot connect to the camera during this time, but if I kill it and recycle power it boots back into the Wyze FW and is working

Any detail would be helpful here, I'm grateful for the hack, the instructions could have a bit more info... 

Can someone confirm that it works on 4.36.0.228?
Also, cannot find the links for the stream, can we add those in the release notes somwhere?

Thanks!

**Comment by FiveLeavesLeft** [comment]
Take a look at this thread: https://github.com/FiveLeavesLeft/WyzeCameraLiveStream/issues/13 

and in particular this comment

 https://github.com/FiveLeavesLeft/WyzeCameraLiveStream/issues/13#issuecomment-840224370

**Comment by teredactle** [comment]
Thank you for posting that, I didn't see that thread.
Checked it out, disabled the firewall on the CentOS VM I installed for this, then it started loading up and saw the GET statement.

I can also confirm there are no prompts of any kind on the Wyze Cam 3 (begin, finished, etc...).

Having said that, I cannot view the stream from my windows VLC, turned off firewall thinking it could be blocking but no dice. Also, Blue Iris see the camera when I add it (i.e. changed port to 12345) it adds it, but then it cannot connect to it. I don't know if the syntax is correct that it detects, I tried both RTSP and HTTP from both BI and VLC but no stream...

Ports are open, I did a ports can on the IP, both 23 and 12345 are open.
